pyproject.toml

```diff
@@ -39,7 +35,15 @@ name = "preliz"
 source = "https://github.com/arviz-devs/preliz"
 tracker = "https://github.com/arviz-devs/preliz/issues"
 
-
+[project.optional-dependencies]
+full = [
+  "matplotlib>=3.5",
```

Comment:
Shouldn't matplotlib be listed as part of the regular dependencies? Also currently matplotlib will be installed anyway as we have arviz as a dependency

Reply:
Yes